### PR TITLE
Add StarLeaf.app v2.11.32

### PR DESCRIPTION
**Pull request by witsch**
After making all changes to the cask:

- [x] `brew cask audit --download {{cask_file}}` is error-free.
- [x] `brew cask style --fix {{cask_file}}` reports no offenses.
- [x] The commit message includes the cask’s name and version.

Additionally, if **adding a new cask**:

- [x] Named the cask according to the [token reference].
- [x] `brew cask install {{cask_file}}` worked successfully.
- [x] `brew cask uninstall {{cask_file}}` worked successfully.
- [x] Checked there are no [open pull requests] for the same cask.
- [x] Checked the cask was not already refused in [closed issues].
- [x] Checked the cask is submitted to [the correct repo].

[token reference]: https://github.com/caskroom/homebrew-cask/blob/master/doc/cask_language_reference/token_reference.md
[open pull requests]: https://github.com/caskroom/homebrew-cask/pulls
[closed issues]: https://github.com/caskroom/homebrew-cask/issues?q=is%3Aissue+is%3Aclosed
[the correct repo]: https://github.com/caskroom/homebrew-cask/blob/master/doc/development/adding_a_cask.md#finding-a-home-for-your-cask
[version-checksum]: https://github.com/caskroom/homebrew-cask/blob/master/doc/cask_language_reference/stanzas/sha256.md#updating-the-sha256

### CODE REVIEW

**Review by CryoTheRenegade — CHANGES_REQUESTED** [review]
Add Uninstall pkgutil: stanza

**Comment by witsch** [comment]
Thank you, will do... 🙂

**Review by victorpopkov — APPROVED** [review]

File: Casks/starleaf.rb [hunk 1]
```diff
@@ -10,5 +10,7 @@
 
   pkg 'StarLeaf.pkg'
 
-  uninstall pkgutil: 'com.starleaf.breeze2.mac'
+  uninstall login_item: 'StarLeaf',
+            pkgutil:    'com.starleaf.breeze2.mac',
+            quit:       'Starleaf.breeze2'
```
Comment (witsch) [inline Casks/starleaf.rb]: Ah, thanks!  I was wondering about the `Starleaf.breeze2` (in `~/Library/Preferences/Starleaf.breeze2.plist`).  Is there any documentation/readme about the different keys to `uninstall`?

Comment (commitay) [reply Casks/starleaf.rb]: https://github.com/caskroom/homebrew-cask/blob/master/doc/cask_language_reference/stanzas/uninstall.md

**Pull request merged**